packages/core/src/Stores/ui-store.js

```diff
@@ -83,6 +83,9 @@ export default class UIStore extends BaseStore {
     // Remove MX gaming account modal
```

Comment:
Better to update this comment to `MX & MLT` to differentiate it with the one you've added below.